source/__tests__/main.journal.test.js

```diff
@@ -0,0 +1,144 @@
+//main.journal.test.js
+
+import { getCurrentDate, formatDateToYYYYMMDD } from "./date.util.js";
```

Comment:
Same comment as the other branch with updating the import path. Also not sure how the import from `main.journal.js` is being used, but it'd be great if you could add tests for the functions in that file as well.